COSC481W-2024Fall/Ript-Fitness
# Evan/add workouts page

# Add Workouts Page

## Description:

Created the initial Add Workouts Page. This page is for adding workouts to a given Exercise. Then Submitting and saving the workout.

## Key Changes:

- Display a list of Exercise Buttons
- Each button is expandable
- Each button can be rearranged by long pressing and dragging
- Each exercise can be removed by swiping from left to right
- Haptic feedback and an overlay indicates when dragging


## What’s Not Completed:

- A Message should indicate when there are no workouts added to the Exercise yet
- Workout data is currently hardcoded, and needs to be swapped with data from api calls once available.
- Navigation Functionality to/from other workout related screens
- Submit button functionality
- Fix merge conflicts
- Include some information from last time the exercise was ran

## Testing Instructions:

- View Add Workouts Page
- Drag and drop workouts
- Swipe to remove a workout
- Expand a workout
- Press the back button in the top left, and verify page changes

Issue #8
